src/ai/backend/manager/server.py

```diff
@@ -633,6 +634,13 @@ async def server_main(loop: asyncio.AbstractEventLoop,
                 str(root_ctx.local_config['manager']['ssl-cert']),
                 str(root_ctx.local_config['manager']['ssl-privkey']),
             )
+
+        # Start aiomonitor.
+        # Port is set by config (default=50001).
+        m = aiomonitor.Monitor(loop, port=root_ctx.local_config['manager']['aiomonitor-port'])
+        m.prompt = "monitor (manager) >>> "
+        m.start()
```

Comment:
It internally spawns a thread, so we need to call `m.close()` explicitly to properly clean up during shutdown, or use its context manager like `with aiomonitor.start_monitor(...)`.